includes/class-wc-privacy.php

```diff
@@ -0,0 +1,136 @@
+		 *
+		 * Customer Data (meta):
+		 *
+		 * Billing and shipping addresses
+		 *
+		 * Misc:
+		 *
+		 * Downloadable Product User Email
+		 * Download Log Entry User IP Address
+		 * Carts for user ID/Sessions
+		 * File based logs containing their email e.g. from webhooks
+		 * Payment tokens?
+		 *
+		 * Ping tracker to anonomize data there as well.
+		 */
+	}
+
+	/**
+	 * Get personal data for a given email address. This can be used for exports.
+	 *
+	 * @param string $email Email address.
+	 * @return array Array of personal data.
+	 */
+	public static function get_personal_data( $email ) {
+		if ( ! is_email( $email ) ) {
+			return array();
+		}
+
+		$personal_data = array();
+
+		// Check if user has an ID in the DB to load stored personal data.
+		$user = get_user_by( 'email', $email );
+		if ( $user instanceof WP_User ) {
+			$customer = new WC_Customer( $user->ID );
+			if ( $customer ) {
+				$personal_data['Billing First Name']       = $customer->get_billing_first_name();
+				$personal_data['Billing Last Name']        = $customer->get_billing_last_name();
+				$personal_data['Billing Company']          = $customer->get_billing_company();
+				$personal_data['Billing Address 1']        = $customer->get_billing_address_1();
+				$personal_data['Billing Address 2']        = $customer->get_billing_address_2();
+				$personal_data['Billing City']             = $customer->get_billing_city();
+				$personal_data['Billing Postal/Zip Code']  = $customer->get_billing_postcode();
+				$personal_data['Billing State']            = $customer->get_billing_state();
+				$personal_data['Billing Country']          = $customer->get_billing_country();
+				$personal_data['Billing Phone']            = $customer->get_billing_phone();
+				$personal_data['Shipping First Name']      = $customer->get_shipping_first_name();
+				$personal_data['Shipping Last Name']       = $customer->get_shipping_last_name();
+				$personal_data['Shipping Company']         = $customer->get_shipping_company();
+				$personal_data['Shipping Address 1']       = $customer->get_shipping_address_1();
+				$personal_data['Shipping Address 2']       = $customer->get_shipping_address_2();
+				$personal_data['Shipping City']            = $customer->get_shipping_city();
+				$personal_data['Shipping Postal/Zip Code'] = $customer->get_shipping_postcode();
+				$personal_data['Shipping State']           = $customer->get_shipping_state();
+				$personal_data['Shipping Country']         = $customer->get_shipping_country();
+				$personal_data['Shipping Phone']           = $customer->get_shipping_phone();
+			}
+		}
+
+		// Retrieve all orders with billing email set to user's email.
+		$orders = wc_get_orders( array(
+			'billing_email' => $email,
+			'limit'         => -1,
+		) );
+
+		if ( 0 < count( $orders ) ) {
+			$personal_data['orders'] = array();
+			foreach ( $orders as $order ) {
+				$order_data = array(
+					'Transaction ID'           => $order->get_order_number(),
```

Comment:
Order ID being included here could add context.

Also need to get _billing_address_index meta.